lighthouse/helpers/plates.py

```diff
@@ -40,17 +41,33 @@
 from lighthouse.helpers.mysql_db import create_mysql_connection_engine, get_table
 from sqlalchemy.sql.expression import and_  # type: ignore
 from sqlalchemy.sql.expression import bindparam  # type: ignore
+from lighthouse.messages.message import Message  # type: ignore
+from lighthouse.helpers.events import (
+    construct_destination_plate_message_subject,
+    get_robot_uuid,
+    construct_robot_message_subject,
+    construct_mongo_sample_message_subject,
+    construct_source_plate_message_subject,
+    get_message_timestamp,
+)
 
 logger = logging.getLogger(__name__)
 
 
+# TODO - Refactor:
+# * move db calls (MLWH and Mongo) to separate files
+# * consolidate small methods into larger ones if the small methods are not re-used elsewhere
```

Comment:
They do, but I disagree with making methods public for the sake of testing. I'm of the opinion it's best to just test through the public endpoints of a file/class, rather than testing all the private methods individually. That way you get a better sense of the purpose of a file/class

Reply:
This is only a comment that can be disregarded in the future, so not going to make a change here 🙂